app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -1,5 +1,6 @@
 require 'image-inspector-client'
 require 'kubeclient'
+require 'securerandom'
```

Comment:
I think that this is not really needed here, I used SecureRandom before without adding this. Can you check?